cmd/docker/builder_test.go

```diff
@@ -144,7 +145,8 @@ func TestBuildkitDisabled(t *testing.T) {
 	assert.NilError(t, dockerCli.Initialize(flags.NewClientOptions()))
 	dockerCli.ConfigFile().CLIPluginsExtraDirs = []string{dir.Path()}
 
-	tcmd := newDockerCommand(dockerCli)
+	ctx := context.TODO()
```

Comment:
I think we need to use the `fakeClient.Ping()` context here instead of a new context. If we want the `newDockerCommand` to have it's own context with a cancel, we could wrap it with `context.WithCancel(parentCtx)`.